src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        StringBuilder builder = new StringBuilder();
+        builder.append("Figure: the radius is")
+                .append(radius)
+                .append(" the color is: ")
+                .append(color);
+        System.out.println(builder.toString());
+    }
```

Comment:
The `draw()` method should not be responsible for console output. Consider separating the concerns by having the `draw()` method return a string representation of the figure, and let another class or method handle the output.

Reply:
no